### ISSUE TITLE
Issue with "delete"

### ISSUE THREAD

**Issue by rianquinn** [jeff-hykin/better-cpp-syntax#407]
Checklist

- [x] This problem exists even with the setting `"C_Cpp.enhancedColorization": "Disabled"`
- [ ] This bug exists for C
- [x] This bug exists for C++
- [ ] This bug exists for Objective-C
- [ ] This bug exists for Objective-C++

## The code with a problem is:
```
unique_dynarray(const unique_dynarray &) = delete;
unique_dynarray &
operator=(const unique_dynarray &) = delete;
```

## It looks like:
** screenshot + theme name preferable **
![image](https://user-images.githubusercontent.com/5932003/67213566-bd592900-f40d-11e9-8768-31b18bde5668.png)

## It should look like:
The "delete" keyword should be the same color for both

**Comment by matter123** [comment]
This is likely an issue with your color theme.

The delete on line one is correctly recognized and has the scope `keyword.other.delete.constructor`.

The delete on line one is correctly recognized and has the scope `keyword.operator.delete`. (@jeff-hykin is that expected?)

You can use the vscode setting `editor.tokenColorCustomizations` to change the coloring for the tokens.
For more information on how to do this see #7 or the [visual studio code docs](https://code.visualstudio.com/docs/getstarted/themes#_customizing-a-color-theme).

Alternatively, if you feel that this behavior should be fixed by default, you may wish to file an issue with your color-theme.

**Comment by jeff-hykin** [comment]
He is actually correct, there is a highlighting issue
```
class unique_dynarray {
    public:
    unique_dynarray(const unique_dynarray &) = delete;
    unique_dynarray &
    operator=(const unique_dynarray &) = delete;
}
```

![Screen Shot 2019-10-22 at 8 53 38 AM](https://user-images.githubusercontent.com/17692058/67293033-7882d000-f4a9-11e9-82b9-8b57ea0219cb.png)

Both of them should be `keyword.other.delete` (light blue) since they're not operators (purple).

**Comment by MonaTheMonad** [comment]
This seems to arise with `default`, too. If you replace `delete` with `default`, the same colourisation occurs.

**Comment by peaceshi** [comment]
![image](https://user-images.githubusercontent.com/10380828/71323868-e92e8300-2512-11ea-9071-327e9ad1d839.png)
maybe you can try to use my theme. 
https://marketplace.visualstudio.com/items?itemName=peaceshi.syntax-highlight

or added this with the color you like:
``` json
keyword.operator { "fontStyle": "", "foreground": "#f07178" },
keyword.other.delete { "fontStyle": "", "foreground": "#f07178" }
```

**Comment by sean-mcmanus** [comment]
We got a similar report at https://github.com/microsoft/vscode-cpptools/issues/5715, with the additional issue with "default" having scope keyword.control.default.cpp. Repro is 
```
struct Test {
    Test(Test&) = default;
    Test(Test&&) = delete;
    Test& operator=(const Test &) = default;
    Test& operator=(Test&&) = delete;
};
```

**Comment by jeff-hykin** [comment]
Should be fixed now (`v1.15.24` released just now), but let me know if it persists
